### PR TITLE
docs(MAINTAIN):  additional information about the release of the LTS …

### PR DESCRIPTION
**Pull request by tzssangglass**
…version

### Description

Fixes # (issue)

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

**Review by spacewander — APPROVED** [review]

File: MAINTAIN.md [hunk 1]
```diff
@@ -38,15 +37,15 @@ via `VERSION=x.y.z make release-src`
 11. Update APISIX rpm package
     > Go to [apisix-build-tools](https://github.com/api7/apisix-build-tools) repository and create a new tag named `apisix-${x.y.z}` to automatically submit the
     package to yum repo
-12. First, update [APISIX docker](https://github.com/apache/apisix-docker/commit/829d45559c303bea7edde5bebe9fcf4938071601) in [APISIX docker repository](https://github.com/apache/apisix-docker), after PR merged, then create a new branch from master, named as `release/apisix-${version}`, e.g. `release/apisix-2.10.2`
+12. - If the version number is the largest, update [APISIX docker](https://github.com/apache/apisix-docker/commit/829d45559c303bea7edde5bebe9fcf4938071601) in [APISIX docker repository](https://github.com/apache/apisix-docker), after PR merged, then create a new branch from master, named as `release/apisix-${version}`, e.g. `release/apisix-2.10.2`.
```
Comment (tokers) [inline MAINTAIN.md:40]: I don't understand the meaning of `largest`? Do you mean `latest`?

Comment (tzssangglass) [reply MAINTAIN.md:40]: for example, now the `latest` version is 2.13.2, it's an LTS version and released on 27 June, but the `largest` version is 2.14.1 which was released on 27 May. When I wanted to release 2.13.3, the `largest` version is also, but it is not the `latest` version, the `latest` version is  2.13.2.

Comment (tzssangglass) [reply MAINTAIN.md:40]: - in terms of serial number, `2.14.1` is `largest`
- in terms of release time, 2.13.2 is `latest`

Comment (tzssangglass) [reply MAINTAIN.md:40]: ping @tokers @moonming

**Review by bzp2010 — APPROVED** [review]

**Pull request merged**